tools/main.c

```diff
@@ -60,7 +60,7 @@ main (int   argc,
     case 1:
         execute_man (argv[0], envp);
         LOG_ERR ("failed to load manpage, check your environment / PATH\n");
-        /* fallthrough */
+        exit (1);
```

Comment:
IMHO the compiler settings you've got setup right now are correct. My reasoning went something like this: It's a legit fallthrough case but even then the original author (me?) felt it was necessary to document it with a comment. Might as well just fill that line with an explicit call and remove all ambiguity.